build.py

```diff
@@ -2299,3 +2303,11 @@ def enable_all():
             p = subprocess.Popen([f'./{script_name}'], cwd=FLAGS.build_dir)
         p.wait()
         fail_if(p.returncode != 0, 'build failed')
+
+
+# apt get update
+# apt install -y install default-jre
+# apt install -y doctest-dev
+# apt install -y libboost-program-options-dev
+# LD_LIBRARY_PATH=/opt/tritonserver/lib:$LD_LIBRARY_PATH
+# LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
```

Comment:
remove?

Reply:
Also removing "libboost-program-options-dev \" from build.py. It used to be a dependency but after removing many of the unnecessary directories from the modern-cpp-kafka directory in the third_party repo, I don't think it's necessary anymore. Mentioning it just in case.